src/font/system/no_std.rs

```diff
@@ -61,15 +61,24 @@ impl FontSystem {
 
     pub fn get_font_matches<'a>(&'a self, attrs: Attrs) -> Arc<FontMatches<'a>> {
```

Comment:
This function has to get _all_ fonts that match attrs for font fallback to work

Reply:
I'm not familiar enough with this system to make that work; but it looks like there is some interest in this PR from various folks contributing to this project. Maybe somebody can pick up where I've left off?

Reply:
If that is the case, I will close the PR and hopefully a compliant implementation will be done at some point.

Reply:
I just realized; `self.db.query` already takes care of the font fallback here and thus this only needs to return one font going forward. We just need to build up the query so that it's always containing something that matches.

Reply:
`db.query` absolutely does not do font fallback. To do so requires shaping with the default font, then finding missing glyphs, then shaping those with the fallback font list.

Reply:
(Which is what the code in src/font/fallback in cosmic-text does)